### ISSUE TITLE
Testing the set up system

### ISSUE THREAD

**Issue by pietrosperoni** [fairdemocracy/vilfredo-setup#1]
The setup has just been updated. @tommasosperoni can you try to see if you can use it to install Vilfredo?

**Comment by pietrosperoni** [comment]
Any question you can ask @polites75

**Comment by pietrosperoni** [comment]
@coorasse you might also want to look at this

**Comment by coorasse** [comment]
I stopped at the second line. `It assumes that you run Debian/GNU Linux version Jessie (8.0 or 8.3).`. I don't have it so I think I cannot install it.

If I proceed I see stuff like: `Note: All the following commands have to be executed as user root.` ARE YOU CRAZY?!? Really?!?

If I go further it becomes even more crazy...`The following example assumes an empty partition is available at /dev/sda3 and three volumes have to be created:`.

This is a setup which tides an application completely to an Operating System and the partitioning of an hard disk. I have never seen something like that...really.

Sorry to be so rude but this setup guide is really really really funny. I'm not able and I'll never be able to install something like that.

I can help for the project when the setup guide will be the following:

```
git clone git@github.com:fairdemocracy/vilfredo.git
cd vilfredo
./bin/setup
start the server
```

**Comment by coorasse** [comment]
Can't you provide a docker instance???

**Comment by pietrosperoni** [comment]
Thanks for the feedback, @coorasse . Yes for now Vilfredo is very difficult to install. In fact it is good news that we are able to install it at all :-). Part of the work necessary is to make it more portable.

<img width="679" alt="screen shot 2016-04-28 at 11 04 46" src="https://cloud.githubusercontent.com/assets/1333674/14882351/3cd5e0b0-0d30-11e6-8d37-09f591650272.png">

That's a claim I will consider :-)

**Comment by polites75** [comment]
> I stopped at the second line. |It assumes that you run Debian/GNU Linux version Jessie (8.0 or 8.3).|. I don't have it so I think I cannot install it.
> If I proceed I see stuff like: |Note: All the following commands have to be executed as user root.| ARE YOU CRAZY?!? Really?!?
> If I go further it becomes even more crazy...|The following example assumes an empty partition is available at /dev/sda3 and three volumes have to be created:|.
> This is a setup which tides an application completely to an Operating System and the partitioning of an hard disk. I have never seen something like that...really.
That guide was originally oriented to Pietro and Derek to be able to reinstall Vilfredo on a brand new virtual machine as I did. So I just listed all steps I followed.
Our first virtual machine was hosted on Cloudatcost and required repartitioning to use all the available disk space. This is not normally needed on other servers.
The system can be installed as well on Ubuntu (has been tested by Derek).
It can coexist with other web applications thanks to use of virtual environments.
The uwsgi-pypy package has been designed to be used also on a system where the original uwsgi package has been installed (all binaries and folders have been renamed).
I'll later reorganize the setup guide separating the case you need to start from scratch on an empty virtual machine or you're installing on an existing server.
See you

Massimiliano

**Comment by pietrosperoni** [comment]
Thanks @polites75 , I will open an instance with this:

> I'll later reorganize the setup guide separating the case you need to start from scratch on an empty virtual machine or you're installing on an existing server.

**Comment by pietrosperoni** [comment]
> Can't you provide a docker instance???

What is a docker instance? In case open an issue, asking for it.

**Comment by giovanisp** [comment]
+1 for Docker, if someone can have the time to set it up. We should focus on making it work damn right on the main functions to ship a Minimum Viable Product, quote:

> Steve Blank: "Tu stai vendendo la visione e consegnando il minimo insieme di funzionalità ai visionari, non a tutti."

**Comment by tommasosperoni** [comment]
I can try to set up a docker instance if you are still interested!

**Comment by giovanisp** [comment]
Cool. @coorasse would you use a docker instance if we could make it and give us some feedback? Thanks.

**Comment by coorasse** [comment]
Yes. Of course! This would make the setup possible also on my laptop.

Alessandro Rodi

**Comment by pietrosperoni** [comment]
This is great, thanks!

**Comment by giovanisp** [comment]
@tommasosperoni please let us know if you need information or help to go ahead ;-)

**Comment by pietrosperoni** [comment]
Also @olistik approved the idea of the docker. Looking forward for it.
I spoke with @tommasosperoni just yesterday evening. He said he was going to send you a mail, @giovanisp .
